test/withTheme_test.js

```diff
@@ -0,0 +1,34 @@
+import { expect } from "chai";
+import React from "react";
+
+import { withTheme } from "../src";
+import { createComponent, createSandbox } from "./test_utils";
+
+describe.only("withTheme", () => {
```

Comment:
```suggestion
describe("withTheme", () => {
```

@mirajp change `describe.only` to `describe` and it should solve the problem you mentioned about tests 😉

Reply:
Never mind, I had to manually call `touch` on the file... Idk how chai/whatever this test infrastructure works but I guess it caches copies of the file and doesn't know it's updated?